dbt/include/bigquery/macros/utils/get_columns_spec_ddl.sql

```diff
@@ -1,18 +1,30 @@
 {% macro bigquery__get_columns_spec_ddl() %}
   {# loop through user_provided_columns to create DDL with data types and constraints #}
-    {%- set ns = namespace(at_least_one_check=False) -%}
+    {%- set ns = namespace(at_least_one_check=False, at_least_one_pk=False) -%}
     {%- set user_provided_columns = model['columns'] -%}
     (
-    {% for i in user_provided_columns -%}
+    {% for i in user_provided_columns %}
       {%- set col = user_provided_columns[i] -%}
-      {% set constraints = col['constraints'] -%}
-      {%- set ns.at_least_one_check = ns.at_least_one_check or col['constraints_check'] %}
-      {{ col['name'] }} {{ col['data_type'] }} {% for x in constraints %} {{ x or "" }} {% endfor %} {{ "," if not loop.last }}
-    {%- endfor %}
-  )
+      {%- set constraints = col['constraints'] -%}
+      {{ col['name'] }} {{ col['data_type'] }}
+      {%- for x in constraints -%}
```

Comment:
I know I mentioned this in the relevant `dbt-core` PR, but I really wish this were part of the column properties. Then we could just add it to line 9 (replacing lines 10-18):
```sql
{{ col['name'] }} {{ col['data_type'] }} {{ col['constraints_sql'] }}
```